### PR TITLE
pot/potluck status update

### PR DESCRIPTION
**Pull request by hny-gd**

### CODE REVIEW

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2022q3/pot.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Links: +
+link:https://github.com/bsdpot[Pot organization on github] URL: link:https://github.com/bsdpot[https://github.com/bsdpot]
+
+Contact: Luca Pizzamiglio (Pot) <pizzamig@freebsd.org> +
+Contact: Stephan Lichtenauer (Potluck) <sl@honeyguide.eu> +
+Contact: Michael Gmelin (Potman) <grembo@freebsd.org>
+
+Pot is a jail management tool that link:https://www.freebsd.org/news/status/report-2020-01-2020-03/#pot-and-the-nomad-pot-driver[also supports orchestration through Nomad].
+
+During the last quarter, link:https://github.com/bsdpot/pot/releases/tag/0.15.3[pot 0.15.3] was released. It contains a number of improvements like mount-out, signal and exec commands, better jail lifecycle handling and many bug fixes.
```
[the remark this answers is not in the document]
Comment (hny-gd) [reply 2022q3/pot.adoc:12]: Added in the new commit and added comma.

File: 2022q3/pot.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+A new version of the Nomad driver link:https://github.com/bsdpot/nomad-pot-driver/releases/tag/v0.9.0[nomad-pot-driver 0.9.0] was also released with signal and exec support and stability fixes.
+
+Potluck aims to be to FreeBSD and pot what Dockerhub is to Linux and Docker: a repository of pot flavours and complete container images for usage with pot and in many cases nomad.
```
[the remark this answers is not in the document]
Comment (hny-gd) [reply 2022q3/pot.adoc:16]: Fixed ("Nomad")

File: 2022q3/pot.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Since the last status report, link:https://github.com/bsdpot/potluck/commits/master[many changes] have been committed, including many fixes and improvements to core images like link:https://github.com/bsdpot/potluck/tree/master/grafana[grafana], link:https://github.com/bsdpot/potluck/tree/master/postgresql-patroni[postgresql-patroni] or link:https://github.com/bsdpot/potluck/tree/master/loki[loki].
```
[the remark this answers is not in the document]
Comment (hny-gd) [reply 2022q3/pot.adoc]: Changed.

**Review by pauamma — APPROVED** [review]
Needs commas around the nomad-pot link text, but that's minor, so I'll do it while merging.

**Pull request merged**